src/audio/public.h

```diff
@@ -11,22 +11,22 @@ void snd_start_sound_with_shift(s32 soundID, u8 volume, u8 pan, s16 pitchShift);
 void snd_adjust_sound(s32 soundID, u8 volume, u8 pan);
 void snd_adjust_sound_with_shift(s32 soundID, u8 volume, u8 pan, s16 pitchShift);
 void snd_stop_sound(s32 soundID);
-void func_800553F4(void);
+void snd_reset(void);
 void snd_start_sound_raw(s32 soundID, s16 volume, s16 pitchShift, s32 pan);
-AuResult snd_ambient_80055448(s32 arg0);
-AuResult snd_ambient_80055464(s32 arg0, s32 arg1);
-AuResult snd_ambient_quick_fade_out(s32 arg0);
-AuResult snd_ambient_slow_fade_out(s32 arg0, s32 arg1);
-AuResult snd_ambient_8005553C(s32 arg0, s32 arg1);
-AuResult snd_ambient_80055590(s32 arg0, s32 arg1);
-AuResult snd_ambient_800555E4(s32 arg0);
-AuResult snd_ambient_80055618(s32 arg0, s32 arg1);
-AuResult au_ambience_set_volume(s32 arg0, s32 arg1, s32 arg2);
-AuResult au_ambience_disable(s32 arg0);
-AuResult au_ambience_enable(s32 arg0);
-void snd_ambient_80055760(s32 arg0);
-AuResult snd_ambient_800557CC(s32 arg0);
-AuResult snd_ambient_80055848(s32 arg0);
+AuResult au_ambient_load_sound(s32 arg0);
```

Comment:
As a general note, I don't think these should be au_*. The intention of the original naming scheme was for au_ to be reserved for low-level audio code which should not be called from elsewhere in the engine (maps, menus, etc). These snd_amb and snd_song functions are part of the public interface for the audio system, and they usually delegate the work to an appropriate au_ function with error checking. sfx_* and bgm_* are similar and perhaps we should spend some time to hash out a consistent convention. Regardless, I think its important to maintain the distinction between different api "layers" in the audio subsystem.